src/app/pages/quickMaps/quickMaps.service.ts

```diff
@@ -49,6 +53,9 @@ export class QuickMapsService {
     private activatedRoute: ActivatedRoute,
     route: ActivatedRoute,
   ) {
+    void this.dictionariesService.getDictionaries([DictionaryType.MICRONUTRIENTS]).then((dicts: Array<Dictionary>) => {
```

Comment:
In my branch I've changed this so that we store the MicronutrientDictionaryItem rather than just it's id, so some of this might be no longer needed.